ov/profiles.go

```diff
@@ -70,6 +76,7 @@ type ServerProfile struct {
 	BootMode              BootModeOption      `json:"bootMode,omitempty"`         // "bootMode": {},
 	Category              string              `json:"category,omitempty"`         // "category": "server-profiles",
 	Connections           []Connection        `json:"connections,omitempty"`
```

Comment:
connections is an array in connectionSettings struct. Not sure why its present in serverProfile

Reply:
You probably already figured this out since you commented on #119 , but connectionSettings was first introduced in 500.  Before 500 connections was used  - http://h17007.www1.hpe.com/docs/enterprise/servers/oneview3.1/cic-api/en/api-docs/current/index.html#whatIsNew